ansible/roles/deploy-baremetal/templates/kickstart

```diff
@@ -0,0 +1,44 @@
+# Created by Ansible on {{ansible_date_time.date}} at {{ansible_date_time.time}}.
```

Comment:
Our playbooks (and Ansible examples) use `.j2` as templates extension. Emacs recognizes that extension as well.